build.rs

```diff
@@ -137,8 +137,21 @@ fn main() {
             .btree_map(["."])
             .file_descriptor_set_path(protobuf_fds_path);
 
-        tonic_build::configure()
-            .protoc_arg("--experimental_allow_proto3_optional")
+        let mut builder =
+            tonic_build::configure().protoc_arg("--experimental_allow_proto3_optional");
+
+        for path in ["google.rpc.Status"] {
+            builder = builder
+                .type_attribute(path, "#[derive(serde::Serialize)]")
+                .type_attribute(path, "#[serde(rename_all = \"camelCase\")]");
+        }
+
+        // TODO: fix
```

Comment:
skipping it now because the datatype is a bit complex and there is no use of the field. However it would be an improvement at some point

> If the processing of the request fails because the request contains data that cannot be decoded or is otherwise invalid and such failure is permanent, then the server MUST respond with HTTP 400 Bad Request. The Status.details field in the response SHOULD contain a BadRequest that describes the bad data.